### ISSUE TITLE
Paste doesn't work in IE

### ISSUE THREAD

**Issue by tonysparks** [Gbuomprisco/ngx-chips#556]
**Current behavior**
Pasting in IE11 doesn't work (works in Chrome).  The console outputs:
```
ERROR TypeError: Unable to get property 'getData' of undefined or null reference
   "ERROR"
   {
      [functions]: ,
      __proto__: { },
      description: "Unable to get property 'getData' of undefined or null reference",
      message: "Unable to get property 'getData' of undefined or null reference",
      name: "TypeError",
      number: -2146823281,
      stack: "TypeError: Unable to get property 'getData' of undefined or null reference
   at _this.onPasteCallback (http://localhost:4200/vendor.bundle.js:31475:13)
   at Anonymous function (http://localhost:4200/vendor.bundle.js:120431:9)
   at ZoneDelegate.prototype.invokeTask (http://localhost:4200/polyfills.bundle.js:10557:13)
   at onInvokeTask (http://localhost:4200/vendor.bundle.js:99165:21)
   at ZoneDelegate.prototype.invokeTask (http://localhost:4200/polyfills.bundle.js:10557:13)
   at Zone.prototype.runTask (http://localhost:4200/polyfills.bundle.js:10326:21)
   at invoke (http://localhost:4200/polyfills.bundle.js:10621:21)"
   }
```

**Expected behavior**
Pasting values in the `tag-input` works for all browsers.

Seems to be an issue related to this code:
```typescript
    private onPasteCallback = (data: ClipboardEvent): void => {
        const text = data.clipboardData.getData('text/plain');

        text.split(this.pasteSplitPattern)
            .map(item => this.createTag(item))
            .forEach(item => this.onAddingRequested(false, item));

        this.onPaste.emit(text);

        setTimeout(() => this.setInputValue(''), 0);
    }
```

Seems IE likes to be special and requires accessing the clipboard data like so:
```javascript
            var pastedText = "";
            if (window.clipboardData && window.clipboardData.getData) { // IE
                pastedText = window.clipboardData.getData('Text');
            } else if (data.clipboardData && data.clipboardData.getData) {
                pastedText = e.clipboardData.getData('text/plain');
            }         
```

**Minimal reproduction of the problem with instructions (if applicable)**
```html
<div>
    <tag-input       
        id="scheduler-email-input" 
        placeholder=""
        secondaryPlaceholder=""
        name="emails"
        [(ngModel)]="emails"    
        (onAdd)="onAdd($event)"
        (onRemove)="onRemove($event)"
        [attr.required]="true"
        [addOnPaste]="true"
        [pasteSplitPattern]="pasteSplitPattern"
        [addOnBlur]="true"
        [blinkIfDupe]="true"  
        [allowDupes]="false"
        [maxItems]="32"
        [separatorKeys]="[';' , ',' , ' ']"
        [separatorKeyCodes]="[9,13]"
        [theme]="inputClass"
        (onBlur)="onBlur($event)"
        (onFocus)="onFocus($event)"
        (onTextChange)="onTextChange($event)"
        [onAdding]="onAdding"
        #input      
        >

            <template let-item="item" let-index="index">
                <div style="width:100%;">
                    <div class="tag-label"><div class="tag-label-text"
                        popover="{{ item.value }}" 
                        placement="top"
                        triggers="mouseenter:mouseleave"
                        container="body">{{ item.value }}</div></div>
                    <i class="fa fa-times tag-remove-label ng2-tag__remove-button pull-right"
                        (click)="input.removeItem(item, index)"></i>
                </div>
            </template>
    </tag-input>
</div>
```

**What do you use to build your app?. Please specify the version**
angular-cli:1.3.1

**Angular version:** 
angular:4.1.2

**ngx-chips version:**
ngx-chips:1.5.3

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
IE10/IE11

**Comment by stevenrobijns** [comment]
I can confirm this bug.

**Comment by Gbuomprisco** [comment]
Hi, fixed in v 1.7.4
